scripts/reversim-calendar/agenda.js

```diff
@@ -0,0 +1,343 @@
+const axios = require("axios");
+const halls = ['A10', 'A3', 'A4', 'A5'];
```

Comment:
I would add a comment that `halls`, `day1` and `day2` are copied from this and that files.